Example/Okta/AuthViewController.swift

```diff
@@ -22,14 +22,14 @@ class AuthViewController : UIViewController {
         return ProcessInfo.processInfo.environment["UITEST"] == "1"
     }
     
-    private var testConfig: [String: String] {
-        return [
+    private var testConfig: OktaAuthConfig {
+        return try! OktaAuthConfig(with: [
```

Comment:
If any of these values are undefined, this constructor will cause the application to crash. Probably OK because this is for _testing_, but ideally we'll have proper null checking in place.

Reply:
Done.